c++/src/kj/memory.h

```diff
@@ -594,6 +607,13 @@ Own<Decay<T>> heap(T&& orig) {
   return Own<T2>(new T2(kj::fwd<T>(orig)), _::HeapDisposer<T2>::instance);
 }
 
+template <typename T, void(*F)(T*)>
+Own<T> rawPtr(T* ptr) {
```

Comment:
Bikeshedding, but `rawPtr` doesn't quite feel like the right name to me. The important bit is that we're deriving a disposer from a function. Something like `disposedWith()`? I dunno.

Reply:
Yeah I intended to revisit after getting initial feedback. I don't think it's right either.

I can't think of anything better than `disposeWith` or `disposedWith`. I think I slightly prefer `disposeWith`.